src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,23 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed === true,
+    })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user && <UserInfo user={user} key={user.id} />}
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
there is no `map` here, so you don't need to specify key